www/_template/guides/plugins.md

```diff
@@ -120,6 +119,31 @@ The `load()` method is responsible for loading and build files from disk while t
 
 **See it in action:** Let's say that we have a source file at `src/components/App.jsx`. Because the `.jsx` file extension matches an extension in our plugin's `resolve.input` array, Snowpack lets this plugin claim responsibility for loading this file. `load()` executes, Babel builds the JSX input file from disk, and JavaScript is returned to the final build.
 
+
+You could use this method to write a plugin to parse Markdown (`.md`), Nunjucks (`.njk`) or templating languages into HTML. Here is an example that parses Markdown into HTML:
+
+```js
+const { parse } = require('markdown-wasm/dist/markdown.node.js');
+const fs = require('fs');
+
+module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
+  return {
+    name: 'my-markdown-parser',
+    resolve: {
+      input: ['.md', '.markdown'],
+      output: ['.html'],
+    },
+    load: ({filePath}) => {
+      const markdown = fs.readFileSync(filePath, 'utf-8');
+      const code = parse(markdown, pluginOptions);
+      return {
+        '.html': code,
```

Comment:
👍🏻  In general this is a good pattern to specify the file extension in `return ` like this. Even though you can omit it if you’re not changing the file type, I think it’s helpful to be explicit in `load()`.

Also hopefully it separates `load()` from `transform()` in that you can change the extension in `load()` but not `transform()`.